Backtesting/collectors/exchanges/exchange_history_collector/history_collector.py

```diff
@@ -85,19 +86,22 @@ async def start(self):
 
                 for time_frame_index, time_frame in enumerate(self.time_frames):
                     self.current_step_index = (symbol_index * len(self.time_frames)) + time_frame_index + 1
+                    #if self.should_stop:
+                        #return
```

Comment:
we should remove this debug code before merging :)

Reply:
ah yes ! X)